src/schema/sources.ts

```diff
@@ -822,4 +836,7 @@ export const resolvers: IResolvers<any, Context> = {
   Source: {
     permalink: (source: GQLSource): string => getSourceLink(source),
   },
+  SourceMember: {
+    roleRank: (member: GQLSourceMember) => roleRank[member.role],
+  },
```

Comment:
I have introduced this to make it easier to decide in the Frontend whether to show the context menu option of "Remove member".

Reply:
This would return a string right?
The type you described states `Int`.

Do we need to know the string value though?

Reply:
No, this would return an `Int` as shown in the snapshot.

Reply:
It doesn't feel right to me to "leak" this information and rely on the frontend.
I'd suggest having a field `canRemoveMember` and the frontend will use this.
Or an alternative would be to return an array of allowed actions. This might even be a more flexible solution for future actions